libraries/AP_NavEKF2/AP_NavEKF2_Measurements.cpp

```diff
@@ -584,15 +584,26 @@ void NavEKF2_core::readGpsData()
 
             // see if we can get origin from frontend
             if (!validOrigin && frontend->common_origin_valid) {
-                setOrigin(frontend->common_EKF_origin);
+
+                if(!setOrigin(frontend->common_EKF_origin)) {
```

Comment:
nitpick: can you add a space between the "if" and the bracket?  sorry, i know I'm being annoying..

Reply:
fixed. no problem